submodules/islandora_webform_ingest/includes/utilities.inc

```diff
@@ -507,8 +507,13 @@ function iwi_set_form_element_byhash(&$form, &$form_state, $elementspath, $input
                 if (!isset($special_element_children[$offset])) {
                   // If not already cloned for this offset clone it.
                   $new_tab = clone $tab;
-                  $set_element_properties = function ($element) {
+                  $set_element_properties = function ($element) use ($special_xmlform_elements) {
                     $element->default_value = NULL;
+                    foreach($element->children as $key => &$child) {
+                      if($key !== 0 && in_array($child->controls['#type'], $special_xmlform_elements) ) {
```

Comment:
This is meant to address https://github.com/commonmedia/islandora_webform/issues/27. We only are trying to clone the structure of the descendent special xmlform elements, not their data. So there is no reason to keep more than one.